.github/workflows/lint.yml

```diff
@@ -87,6 +87,46 @@ jobs:
         run: |
           (! git --no-pager grep -I -no $'#include <cub/' --  ./aten  ':(exclude)aten/src/ATen/cuda/cub.cuh' || (echo "The above files have direct cub include; please include ATen/cuda/cub.cuh instead and wrap your cub calls in at::native namespace if necessary"; false))
 
+  clang-format:
+    runs-on: ubuntu-18.04
+    if: github.event_name == 'pull_request'
```

Comment:
only apply to `pull_request`, the same setup as the original workflow.

the script `tools/linter/clang_format_ci.sh` depends on the git diff, and only checks the diff